src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -127,6 +127,13 @@ class BladeCompiler extends Compiler implements CompilerInterface
      */
     protected $classComponentNamespaces = [];
 
+    /**
+     * The array of handlers objects should be passed through.
+     *
+     * @var array
+     */
+    public static $echoHandlers = [];
```

Comment:
Does this have to be a static property? Isn't the `BladeCompiler` registered as singleton? 🤔